js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillPage.tsx

```diff
@@ -10,42 +9,28 @@ import {
   Tab,
   Tabs,
 } from '@dagster-io/ui-components';
-import dayjs from 'dayjs';
-import duration from 'dayjs/plugin/duration';
-import relativeTime from 'dayjs/plugin/relativeTime';
-import {useContext, useEffect, useReducer} from 'react';
+import {useContext} from 'react';
 import {Link, useParams} from 'react-router-dom';
-import styled from 'styled-components';
 
-import {BACKFILL_ACTIONS_BACKFILL_FRAGMENT, BackfillActionsMenu} from './BackfillActionsMenu';
+import {BackfillActionsMenu} from './BackfillActionsMenu';
+import {BackfillAssetPartitionsTable} from './BackfillAssetPartitionsTable';
 import {BackfillLogsTab} from './BackfillLogsTab';
-import {BackfillPartitionsTab} from './BackfillPartitionsTab';
+import {BackfillOverviewDetails} from './BackfillOverviewDetails';
 import {BackfillRunsTab} from './BackfillRunsTab';
-import {BackfillStatusTagForPage} from './BackfillStatusTagForPage';
-import {TargetPartitionsDisplay} from './TargetPartitionsDisplay';
-import {
-  BackfillStatusesByAssetQuery,
-  BackfillStatusesByAssetQueryVariables,
-} from './types/BackfillPage.types';
+import {useBackfillDetailsQuery} from './useBackfillDetailsQuery';
 import {CloudOSSContext} from '../../app/CloudOSSContext';
-import {PYTHON_ERROR_FRAGMENT} from '../../app/PythonErrorFragment';
 import {PythonErrorInfo} from '../../app/PythonErrorInfo';
 import {QueryRefreshCountdown, useQueryRefreshAtInterval} from '../../app/QueryRefresh';
 import {useTrackPageView} from '../../app/analytics';
-import {Timestamp} from '../../app/time/Timestamp';
 import {BulkActionStatus} from '../../graphql/types';
 import {useDocumentTitle} from '../../hooks/useDocumentTitle';
 import {useQueryPersistedState} from '../../hooks/useQueryPersistedState';
 import {
   DaemonNotRunningAlert,
   useIsBackfillDaemonHealthy,
 } from '../../partitions/BackfillMessaging';
-import {useBlockTraceOnQueryResult} from '../../performance/TraceContext';
 import {testId} from '../../testing/testId';
 
-dayjs.extend(duration);
-dayjs.extend(relativeTime);
```

Comment:
I think these didn't need to be here because formatDuration in this file no longer uses dayJS